cmd/checks_windows.go

```diff
@@ -353,6 +353,8 @@ func userDetail(userName, detailName, detailValue string) (bool, error) {
 		return user.IsAdmin == lookingFor, nil
 	case "PasswordNeverExpires":
 		return user.PasswordNeverExpires == lookingFor, nil
+	case "NoChangePassword:
```

Comment:
Missing a closing quote :(

Reply:
it somehow managed to compile without the closing quote, I have no idea how that happened